provider-utils/awscloudformation/cloudformation-templates/vod-helpers/LambdaFunctions/InputLambda/index.js

```diff
@@ -0,0 +1,84 @@
+
+// Load the AWS SDK for Node.js
+// eslint-disable-next-line import/no-extraneous-dependencies
+/* eslint-disable */
+const AWS = require('aws-sdk');
+/* eslint-enable */
+const jobSettings = require('./settings.json');
+// Set the region
+
+exports.handler = async (event) => {
+  AWS.config.update({ region: event.awsRegion });
+  const s3 = new AWS.S3({ apiVersion: '2006-03-01' });
+  // Object was deleted
+  if (event.Records[0].eventName === 'ObjectRemoved:Delete') {
+    // Grabbing Key name and Bucket name from event
+    const DeletedKey = event.Records[0].s3.object.key;
+    const split = DeletedKey.split('.')[0];
+    const outputBucketName = process.env.OUTPUT_BUCKET;
+
+    // Call S3 to obtain a list of the objects in the bucket
+    const res = await s3.listObjectsV2({
+      Bucket: outputBucketName,
+      Prefix: split,
+    }).promise();
+
+    // Clean up response to submit to deleteObjects API
+    if (res.Contents.length === 0) {
+      return;
+    }
+    const objectsToDelete = res.Contents.map(x => ({ Key: x.Key }));
+    // Delete the listed objects with prefix of the key that was deleted.
+    await s3.deleteObjects({
+      Bucket: outputBucketName,
+      Delete: {
+        Objects: objectsToDelete,
+        Quiet: false,
```

Comment:
Quiet should be set to true per the docs if we want to use it.

https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/S3.html#deleteObjects-property

https://docs.aws.amazon.com/AmazonS3/latest/API/API_DeleteObjects.html
^ this is what quiet mode does.